app/com/linkedin/drelephant/ElephantContext.java

```diff
@@ -105,19 +113,38 @@ public Configuration getAutoTuningConf() {
     return _autoTuningConf;
   }
 
+  public Properties getElephnatConf() {
+    return elephantProperties;
+  }
+
   private void loadConfiguration() {
     loadAggregators();
     loadFetchers();
     loadHeuristics();
     loadJobTypes();
     loadGeneralConf();
     loadAutoTuningConf();
-
+    loadElephantConf();
```

Comment:
Shouldn't we put the property in GeneralConf . Why to load one more file for just one configuration